autoPlay.user.js

```diff
@@ -1251,13 +1292,18 @@ function tryUsingAbility(abilityId) {
 	return true;
 }
 
-function tryUsingItem(itemId) {
-	if (!canUseItem(itemId)) {
-		return false;
-	}
-
-	triggerItem(itemId);
-	return true;
+function tryUsingItem(itemId, checkInLane) {
+    if (!canUseItem(itemId)) {
+        return false;
+    }
+    if (typeof checkInLane === "undefined") {
```

Comment:
It's not. It's an optional argument. Otherwise I would have to change every single call to tryUsingItem out there.